includes/challenges-13.test.js

```diff
@@ -0,0 +1,273 @@
+// ------------------------------------------------------------------------------------------------
+
+const allHappy = (strs) => {
+  return strs.every(string => string.includes(':)'));
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 6
+//
+// Write a function named findAnything that takes in an array of strings, along with a target string.
+// Return an array containing only those strings from the original array that contain the target string.
+// ------------------------------------------------------------------------------------------------
+
+const findAnything = (strs, target) => {
+  return strs.filter( string => string.includes(target));
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 7
+//
+// Write a function named allAnything that takes in an array of strings, along with a target string.
+// Return a boolean based on whether or not every string in the array contains the target string.
+// ------------------------------------------------------------------------------------------------
+
+const allAnything = (strs, target) => {
+  return strs.every( string => string.includes(target)); 
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 8
+//
+// We've been testing a new course enrollment system, and we think we have the bugs worked out, but
+// in the meantime, Brook enrolled himself in a bunch of different classes to test if it was working.
+//
+// Write a function named unenrollBrook that takes in a 2d array where each array represents
+// one course's roster, and is an array of strings of the names of the people in that course.
+// Return a 2d array with the same roster, but where anyone whose name includes Brook
+// is removed from every course.
+// ------------------------------------------------------------------------------------------------
+
+const unenrollBrook = (roster) => {
+  return roster.map(arr => arr.filter( string => !string.includes('Brook')));
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 9
+//
+// Write a function named sortByDay that takes in an array of strings, each of which represents an
+// event's day and time.
+//
+// Return a 2d array that organizes those strings based on the day they happen.
+// For example, all Monday's events are in the first array, all Tuesday's are in the second, etc.
+//
+// If an event takes place on multiple days (i.e. "Dancing on Mondays and Tuesdays"), it should
+// appear in both arrays.
+// ------------------------------------------------------------------------------------------------
+
+const daysOfWeek = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
+
+const sortByDay = (eventStrings) => {
+  return eventStrings.reduce( (arr, string) => {
```

Comment:
you should be able to do this using the `daysOfWeek` array to make your code cleaner